app/api/custom/orders.py

```diff
@@ -93,3 +102,72 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
+    for attribute in attendee:
+        attendee[attribute.replace('-', '_')] = attendee.pop(attribute)
+    schema = AttendeeSchema()
+    json_api_attendee = {"data": {"attributes": data['attendee'], "type": "attendee"}}
+    result = schema.load(json_api_attendee)
+    if result.errors:
+        return jsonify(result.errors)
```

Comment:
This will send HTTP status code as 200, which means OK. Which means order was created successfully